framework/platform/bpl/CMakeLists.txt

```diff
@@ -73,6 +74,10 @@ elseif (TARGET_PLATFORM STREQUAL "rdkb")
 
     list(APPEND BPL_LIBS nl-3 nl-route-3 bridge dl swpal uci ubox ubus slibc)
 
+    # Install the UCI DB files
```

Comment:
Should be installed to `${CMAKE_INSTALL_PREFIX}/etc/uci-defaults/` - @arnout please confirm.

Reply:
It should be just plain `/etc` - CMAKE_INSTALL_PREFIX defaults to /usr (or /usr/local, I'm not sure) and is overridden by us to /opt/prplmesh (or /opt/beerocks, I'm not sure). We need plain and simple /etc. Note that the DESTDIR is still prepended to that, i.e. if you install with `make install DESTDIR=/path/to/openwrt/staging` (which is what openwrt/ugw does), it goes to /path/to/openwrt/staging/etc which is exactly where we want it.

*However* /etc/uci-defaults if for *scripts* which *generate* the initial database, based on runtime information (e.g. actual network interfaces on the board). So if we put it there, it should be:

- executable;
- a script that checks if the uci config already exists, and exit if not;
- a filename starting with numbers, e.g. 00_prplmesh.

Reply:
Decision is to leave it as is.